pkg/pillar/cmd/zedagent/handlenetworkinstance.go

```diff
@@ -180,9 +180,11 @@ func prepareAndPublishNetworkInstanceInfoMsg(ctx *zedagentContext,
 	//We queue the message and then get the highest priority message to send.
 	//If there are no failures and defers we'll send this message,
 	//but if there is a queue we'll retry sending the highest priority message.
-	zedcloud.SetDeferred(zedcloudCtx, uuid, buf, size, statusURL,
+	const ignoreErr = LOCDest
+	ditems := createDeferredItemsByDest(ctx, dest, ignoreErr, uuid, buf, size,
 		true, false, zinfo.ZInfoTypes_ZiNetworkInstance)
-	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, true)
+	zedcloud.AddDeferredList(zedcloudCtx, ditems)
+	zedcloud.HandleDeferred(zedcloudCtx, time.Now(), 0, false)
```

Comment:
Why are you changing the last argument to HandleDeferred from true to false here?